src/screens/LiquidityPools/LiquidityPoolDashboard.js

```diff
@@ -191,16 +191,20 @@ const LiquidityPoolDashboard = ({
     ? supportedAssets.find(({ symbol }) => symbol === pool.rewards?.[0].symbol)
     : undefined;
 
-  const balance = poolStats.userLiquidityTokenBalance;
+  const balance = poolStats.userLiquidityTokenBalance.toNumber();
   const formattedBalance = formatTokenAmount(balance, pool.symbol);
   const hasBalance = balance > 0;
 
   const fiatCurrency = baseFiatCurrency || defaultFiatCurrency;
   const fiatBalance = formatFiat(convertUSDToFiat(balance * poolStats.currentPrice, rates, fiatCurrency), fiatCurrency);
-  const stakedAmountInFiat = convertUSDToFiat(poolStats.stakedAmount * poolStats.currentPrice, rates, fiatCurrency);
+  const stakedAmountInFiat = convertUSDToFiat(
+    poolStats.stakedAmount.toNumber() * poolStats.currentPrice,
```

Comment:
Again, is `toNumber()` safe here?

Reply:
This is only a rough calculation, no harm to be done here.